custom/icds_reports/sqldata/exports/dashboard_usage.py

```diff
@@ -186,7 +186,10 @@ def get_excel_data(self):
 
             dashboard_uname_rx = re.compile(r'^\d*\.[a-zA-Z]*@.*')
 
-            usernames = [user['username'] for user in users if dashboard_uname_rx.match(user['username'])]
+            usernames = []
```

Comment:
hey, the new code is also doing the same thing right.

Reply:
The earlier code is bypassing the condition

Reply:
@challabeehyv